spec/tli/tli_spec.rb

```diff
@@ -152,6 +152,23 @@
     end
   end
 
+  describe 'validate languages passed as parameters' do
+    context 'with valid languages' do
+      it 'should open the tli interactive mode' do
+        readline = double('readline')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.